### PR TITLE
Registry test

### PR DESCRIPTION
**Pull request by hithisiskhai**
Rebased from main
Added cors package
Implement couple side add to registry functionality

### CODE REVIEW

**Review by kyle-sablan — APPROVED** [review]
Everything looks good to me!

File: src/registry/registryadd.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+            >
+              add gifts
+            </PlumButton>
+          )}
+        </Col>
+      </Row>
+      <Row>
+        <h2
+          style={{
+            fontSize: '24px',
```
Comment (DylanDKim) [inline src/registry/registryadd.js:84]: not big but for font styles make sure to use the styled text in /common/styled/textstyles/etc

File: src/registry/registryadd.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+            fontFamily: 'Faustina serif',
+          }}
+        >
+          Your Top Gifts
+        </h2>
+        {registry.length === 0 ? (
+          <h3>You have NOOOO gifts in your registry! Try adding some.</h3>
```
Comment (DylanDKim) [inline src/registry/registryadd.js:91]: lol

File: src/registry/registrystart.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Carousel, Container, Row } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import PlumButton from '../Common/styled/buttonstyles/PlumButton';
+
+export default function ResgistryStart() {
```
Comment (DylanDKim) [inline src/registry/registrystart.js:6]: typo?

**Review by DylanDKim — APPROVED** [review]

File: src/registry/registrystart.js (hunk 3 above)
Comment (hithisiskhai) [reply src/registry/registrystart.js:6]: Need to rearrange the pages.

**Pull request merged**